dockstore-webservice/src/main/java/io/dockstore/webservice/api/PrivilegeRequest.java

```diff
@@ -13,7 +13,7 @@ public class PrivilegeRequest {
     private boolean admin;
     private boolean curator;
     private Partner platformPartner;
```

Comment:
Haven't looked in detail at this object. Does it make sense to do validation or reject requests where the platformPartner doesn't match metricsRobotPartner?
Actually do we need both? Proceeding below.

Reply:
Link a follow-up ticket if you don't have an objection. Doesn't block, but feels a little weird that this can represent a request that doesn't make sense.